book/page_creation.rst

```diff
@@ -137,7 +137,7 @@ You can even shorten this with the handy :class:`Symfony\\Component\\HttpFoundat
     // --> don't forget this new use statement
     use Symfony\Component\HttpFoundation\JsonResponse;
 
-    class LuckyController
+    class LuckyController extends Controller
```

Comment:
As this article describes how to create one action. So each code example is based on the previous example. It would be very strange to have the extends part in all examples but one.

Also, as this article is focused on beginners, it's confusion when not extending Controller imo.